.development.env

```diff
@@ -24,3 +24,4 @@ WEB_SERVER_UWSGI_ASYNC_CORES=10
 CDN_URL=https://cdn.eq.census-gcp.onsdigital.uk
 CDN_ASSETS_PATH=/design-system
 ADDRESS_LOOKUP_API_URL=https://whitelodge-ai-api.census-gcp.onsdigital.uk
+COOKIE_SETTINGS_URL=True
```

Comment:
This could be defaulted to `#` so the page doesn't link anywhere external. Will fix the issue Lewis highlighted.

Reply:
Yeah good idea. Ill sort that